pkg/providers/imagefamily/bootstrap/cse_cmd_nbcontract.sh.gtpl

```diff
@@ -132,7 +132,7 @@ CSE_INSTALL_FILEPATH="/opt/azure/containers/provision_installs.sh"
 CSE_DISTRO_INSTALL_FILEPATH="/opt/azure/containers/provision_installs_distro.sh"
 CSE_CONFIG_FILEPATH="/opt/azure/containers/provision_configs.sh"
 CUSTOM_SEARCH_DOMAIN_FILEPATH="/opt/azure/containers/setup-custom-search-domains.sh"
-CUSTOM_SECURE_TLS_BOOTSTRAP_AAD_SERVER_APP_ID=""
+CUSTOM_SECURE_TLS_BOOTSTRAP_AAD_SERVER_APP_ID="{{getCustomSecureTLSBootstrapAADServerAppID .}}"
```

Comment:
we can put this variable into TLS group in the contract too if we haven't yet.

Reply:
ah good catch, it is already in the TLS config but it should be passing in `.TlsBootstrappingConfig`. A question though, why is variable under the section that says `# the following variables should be removed once we set the default values in the Go binary on VHD`? This should be passed from the bootstrapper right? As how RP currently does.

Reply:
discussed offline, was a misunderstanding. `CUSTOM_SECURE_TLS_BOOTSTRAP_AAD_SERVER_APP_ID` is a required field for secure tls bootstrapping that will not be removed.